### ISSUE TITLE
API: Empty reply from server/no reponse 

### ISSUE THREAD

**Issue by alexpovel** [AdguardTeam/AdGuardHome#2687]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * v0.105.0
* **How did you install AdGuard Home:**
  * Docker with `host` network
* **How did you setup DNS configuration:**
  * 
* **If it's a router or IoT, please write device model:**
  * Raspberry Pi 4
* **CPU architecture:**
  * ARM
* **Operating system and version:**
  * Raspbian

### Expected Behavior

As in `v0.104.3` (worked there), API requests are expected to work. As part of an Ansible role, I do the equivalent of:

```bash
curl --location -H "Authorization: Basic <base64 user:password>" -d '{"FilterRefreshRequest": true}' --request POST 'http://localhost/control/filtering/refresh'
```

See also [here](https://github.com/AdguardTeam/AdGuardHome/tree/master/openapi#authentication) and [here](https://github.com/AdguardTeam/AdGuardHome/blob/f68f6c9b37cea1f33cb6fa791e6678b9df3a237b/openapi/openapi.yaml).

### Actual Behavior

API seems to error out the ass at irregular times. `curl` gives `curl: (52) Empty reply from server`, exit code 52. Updating the blocklists from the web interface can also give `Error: control/filtering/refresh | Network Error` (red popup bottom right corner).

**However**, refreshing still works (tested with an editable pastebin) in both cases, the server reply is just malformed.

In the special case of the Ansible role, this means Ansible fails at that point (or retries if configured, then fails), despite succeeding. If it helps, the error message there is `Status code was -1 and not [200]: Connection failure: Remote end closed connection without response` (so same stuff).

**Comment by ainar-g** [comment]
Possible duplicate of #2671?  We've released the new v0.105.1, which should address the issue.

**Comment by alexpovel** [comment]
Yes, it's a duplicate of that, my search terms didn't catch that, sorry.
